src/Controller/Api/BookController.php

```diff
@@ -0,0 +1,139 @@
+<?php
```

Comment:
So here is the place where FOS Bundle starts messing with you.

Reply:
It seems it was a bad idea to use FOS package :)